lib/validataion/jsonSchemas.js

```diff
@@ -365,6 +379,15 @@ schemas[validationKeys.ENV_VARS_INTERACTIVE] = {
 	},
 };
 
+schemas[validationKeys.ARRAY_OF_BUTTONS] = {
```

Comment:
Could be generalized to ARRAY_OF_STRING or non empty strings

Reply:
Added button validation